docs/examples/merging/updates/ghana_tender1.json

```
@@ -0,0 +1,45 @@
+{
+  "version": "1.1",
+  "publishedDate": "2020-10-21T13:02:26Z",
+  "publisher": {
+    "name": "Public Procurement Authority",
+    "scheme": "GH-RGD",
+    "uid": "1234 ",
+    "uri": "https://egovonline.gegov.gov.gh/RGDPortalWeb/portal/RGDHome/eghana.portal"
+  },
+  "uri": "https://www.ghaneps.gov.gh/ocds/services/releasepackage/getfullreleasepackageforocid/ocds-uhveoc-144560",
+  "releases": [
+    {
+      "ocid": "ocds-uhveoc-144560",
+      "id": "ocds-uhveoc-144560-1601380494627-tender",
+      "date": "2020-10-21T13:02:26Z",
+      "tag": [
+        "tender"
+      ],
+      "initiationType": "tender",
+      "parties": [
+        {
+          "name": "ECONOMIC AND ORGANISED CRIME OFFICE",
+          "id": "14957",
+          "roles": [
+            "buyer",
+            "procuringEntity"
+          ]
+        }
+      ],
+      "tender": {
+        "id": "144560",
+        "title": "Toners",
+        "description": "The E.O.C.O is requesting for quotations from your outfit for the supply of ten (10) pieces each of 26A and 29A toners.\r\nYou are requested to submit your quotation via GHANEPS portal.\r\nDeadline for submission is 30th October, 2020. at 16:00 HRS GMT.",
```

Remove content that belongs in other fields (`tender.items`, `tender.submissionMethodDetails` and `tender.tenderPeriod.endDate`).

```suggestion
        "description": "The E.O.C.O is requesting quotations for the supply of toners.",
```